sources/engine/Stride.Engine/Profiling/DebugTextSystem.cs

```diff
@@ -64,14 +88,20 @@ public void Print(string message, Int2 position, Color4 color)
         /// </summary>
         public Color4 TextColor { get; set; } = Color.LightGreen;
 
+        /// <summary>
+        /// Sets or gets the time that messages will stay on screen by default.
+        /// </summary>
+        public TimeSpan DefaultOnScreenTime { get; set; } = TimeSpan.FromSeconds(1);
```

Comment:
I think this should be set to zero to behave in the same way that it was before for messages without any time specified

Reply:
Makes sense, i edited it to be TimeSpan.Zero